packages/playwright-test/package.json

```diff
@@ -33,6 +33,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@types/node": "*",
+    "azure-devops-node-api": "^11.2.0",
```

Comment:
Maybe could be a optional peer dependency?

Reply:
if it's better, i can do it